specs/mgt-file-picker.md

```diff
@@ -9,49 +9,41 @@ The file picker component provides a button, a drop down list of files, and as w
+| On selection of a file, a `selectionChanged` event is fired | P0 | |	
 | The selected file object(s) is accessible by the developer for further action |P0 | |	
-| Ability for developer to toggle between single-select and multiselect | P1 |  | 
+| Extended file picker is launched when “See all” is clicked | P0 | |
+| Ability for developer to toggle between single-select and multiselect | P1 | Will not be included in v1. | 
 | Visual indication of selected files | P2 | Will not be included in v1. |
 | Render the selected files | P2 | Will not be included in v1. We will provide a sample for developers to render selected files using `mgt-file` components themselves. |
-| Extended file picker is launched when “See all” is clicked | P2 | Not included in v1. |
 
 ## Proposed Solution
 
 ### Example 1: Show the user's top 10 most recently used files to select from
 ```<mgt-file-picker insight-type=“used” show-max=“10”></mgt-file-picker>```
 
-### Example 2: Show a specific folder in OneDrive to select files from
-```<mgt-file-picker drive-id=“123” item-id=“456”></mgt-file-picker>```
-
 ## Attributes and Properties
 
 | Attribute | Property | Description |
 | --------- | -------- | ----------- |
-| `file-list-query` | `fileListQuery` | The full query or path to the folder in a drive or site to retrieve the list of files from |
-| `file-queries` | `fileQueries` | An array of file queries to be rendered by the component |
 | `files` | `files` | An array of files to get or set the list of files rendered by the component. Use this to access the files loaded by the component. Set this value to load your own files. |
 | `insight-type` | `insightType` | Set to show the user’s trending, used, or shared files. |
-| `drive-id` | `driveId` | Id of the drive the folder belongs to. Must also provide either `item-id` or `item-path`. |
-| `group-id ` | `groupId` | Id of the group the folder belongs to. Must also provide either `item-id` or `item-path`. |
-| `site-id` | `siteId` | Id of the site the folder belongs to. Must also provide either `{item-id}` or `{item-path}`. Provide `{list-id}` if you’re referencing a file from a list. |
-| `item-id` | `itemId` | Id of the folder. Default query is `/me/drive/items`. Provide `{drive-id}`, `{group-id}`, `{site-id}`, or `{user-id}` to query a specific location. |
-| `item-path` | `itemPath` | Item path of the folder. Default query is `/me/drive/root`. Provide `{drive-id}`, `{group-id}`, `{site-id}`, or `{user-id}` to query a specific location. |
 | `show-max` | `showMax` | A number value to indicate the maximum number of files to show. |
-| `selected-files` | `selectedFiles`| An array of selected file objects. |
-| `multiselect` | `multiselect` | Enable/Disable selection of more than 1 file. Default is `false`.|
 
 ## Events
 
 | Event | Description |
 | ----- | ----------- |
-|selectionChanged | Fired when the user selects or deselects a file |
+|selectionChanged | Fired when the user selects a file |
```

Comment:
I propose we rename this to something along the lines of `filesPicked` or `filesSelected`

Reply:
Done